fiftyone/plugins/skills.py

```
@@ -0,0 +1,232 @@
+    central_base = _CENTRAL_GLOBAL if global_ else _CENTRAL_LOCAL
+    central_dir = os.path.realpath(os.path.expanduser(central_base))
+
+    installed = []
+
+    for skill in skills:
+        skill_central = os.path.join(central_dir, skill.name)
+        os.makedirs(skill_central, exist_ok=True)
+
+        dest = os.path.join(skill_central, os.path.basename(skill.path))
+        shutil.copy2(skill.path, dest)
+        logger.info(f"Copied skill '{skill.name}' to '{skill_central}'")
```

Ruff (0.15.7)

[warning] Logging statement uses f-string

(G004)